src/components/header/header.tsx

```diff
@@ -32,16 +34,12 @@ const StyledHeader = styled.header<{
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 0 16px;
+  padding: 0 24px;
 
   border-bottom: ${(props) =>
-    props.$lightTheme
+    props.$lightTheme && !props.$transparent
```

Comment:
I don't understand the naming convention here with the leading dollar sign. It was there before for $lightTheme can someone please help explaining ?